templates/openstackplaybookgenerator/bin/create-playbooks.sh

```diff
@@ -132,5 +147,19 @@ ansible.write_default_ansible_cfg(
 
 EOF_PYTHON
 
-# copy to our persistent volume mount so it shows up in the 'openstackclient' pod
-cp -a /home/cloud-admin/ansible/* /var/cloud-admin/ansible/
+TMP_DIR=$(mktemp -d)
+git clone $GIT_URL $TMP_DIR
+pushd $TMP_DIR
```

Comment:
without popd, wouldn't this end up with the dirs stack growing up trashed with tempdirs?

Reply:
The entire pod is ephemeral and is destroyed (we are using emptydir for local storage with these pods). So I think its okay